.gitignore

```diff
@@ -1,3 +1,4 @@
+*~
```

Comment:
Ignoring your editor's temporary files is more of a per-user issue than something specific to this repository.  I'd recommend you include this in your per-user `~/.gitignore`.